config/api.yml

```diff
@@ -540,6 +540,19 @@
         :identifier: instance_guest_restart
       - :name: reset
         :identifier: instance_reset
+  :notifications:
+    :description: User's past notifications
+    :options:
+    - :collection
+    - :arbitrary_resource_path
+    :verbs: *gp
+    :klass: NotificationRecipient
+    :collection_actions:
+      :post:
+      - :name: mark_as_seen
```

Comment:
no identifiers (authorizations) for mark_as_seen  ?  is authorization essentially being the recipient of the notification ?  
Also, I don't see DELETE,  how are these zapped ?  automatically once seen ?  Thanks.